src/main/java/me/totalfreedom/totalfreedommod/banning/BanManager.java

```diff
@@ -30,13 +30,14 @@
     private final Map<String, Ban> ipBans = Maps.newHashMap();
     private final Map<String, Ban> nameBans = Maps.newHashMap();
     private final List<String> unbannableUsernames = Lists.newArrayList();
+    public static final String CONFIG_FILENAME = "bans.yml";
```

Comment:
This should be being pulled from BanManager.java so as to reduce the likelihood of typos.